src/constants/markets.ts

```diff
@@ -219,3 +220,30 @@ export const ComingSoonMarkets: {[address: string]: ComingSoonMarketsData} = {
     isComingSoon: true,
   },
 }
+
+export type MarketDescriptionData = {
+  IndexesConstruction: Array<string>
+  FullLogo: string
+  ProposedBy: string
+  Approved: string
+  Contract: string
+  Sources: Array<string>
+  AdditionalInfo: string
+}
+
+export const MarketDescription: {[address: string]: MarketDescriptionData} = {
+  '111': {
+    IndexesConstruction: ['Nike Blahs', 'AirJordan 6', 'ASICS Retro', 'Puma Attack', 'Hoka Speedgoat'],
+    FullLogo: BtcdFull,
+    ProposedBy: '@unthoughtfulwizard',
+    Approved: '11.5.24 - OGP 28',
+    Contract: '0xbb12...12J9',
```

Comment:
```suggestion
    Contract: {link: 'https://arbiscan.io/.....'},
```
the contract address is the same as the key of the object, so unnecesary to put here. maybe just store the link here
make it clickable on the build page and direct to link